xcode/README.md

```diff
@@ -7,6 +7,8 @@ targets, two of which are the most likely to be of interest:
 
 * CapstoneStatic, producing `libcapstone.a`, Capstone as a static library;
 * CapstoneDynamic, producing `libcapstone.dylib`, Capstone as a shared library;
+* CapstoneOSXKernel, producing `libCapstoneOSXKernel.a`, Capstone as a static
```

Comment:
i see, but this is inconsistent with the way we are compiling: no matter what options are enable, we always generate the same library name.